pkg/apis/serving/v1alpha1/service_lifecycle.go

```diff
@@ -85,6 +85,13 @@ func (ss *ServiceStatus) PropagateConfigurationStatus(cs *ConfigurationStatus) {
 	}
 }
 
+// MarkRevisionNameTaken notes that the Route has not been programmed because the revision name is taken by a
+// conflicting revision definition.
+func (ss *ServiceStatus) MarkRevisionNameTaken(name string) {
+	serviceCondSet.Manage(ss).MarkUnknown(ServiceConditionRoutesReady, "RevisionNameTaken",
```

Comment:
is this unknown or false? Since a user action is required to recover from this?

Reply:
I think you are right, this should be `MarkFalse`.